doc/dftb+/manual/dftbp.tex

```diff
@@ -3028,7 +3028,10 @@ \subsubsection{Generalized Born Model}
 
 To use the generalized Born model in the SCC procedure use the
 \is{GeneralizedBorn\cb} method in the input to \kw{Solvation}.
-Note that the GB model implemented is only available for finite
+The non-polar solvent area model can be combinded with the GB model
+enabling to additionally correct for hydrogen bonding, the resulting
+model is called GBSA.
+Note that the GB(SA) model implemented is only available for finite
```

Comment:
Probably also a good idea to add a line giving the location of the [repository](https://github.com/grimme-lab/gbsa-parameters) and explaining the state of fitting (since the regression tests and get_external are using data from there).

Reply:
Something like:
Parameters for the model, as fitted for the xtb GFN hamiltonians, are available \href{https://github.com/grimme-lab/gbsa-parameters}{online}.

Reply:
Having the parameters on `dftb.org` as well would also be a nice addition.